.env.example

```diff
@@ -1 +1,5 @@
-PORT=YOUR-PORT
\ No newline at end of file
+PORT=YOUR-PORT
+DEV_DB_URL=[postgres://team7:123456@localhost:5432/balanty]
```

Comment:
Good job Heba!
It is better if you write the URL in general cases not adding your database URL 
`DEV_DB_URL=[postgres://username:userpassword@localhost:5432/databaseName]`